diff --git a/src/libxcrypt/libxcrypt-4.4.28/configure.ac b/src/libxcrypt/libxcrypt-4.4.28/configure.ac
index b9394d87..8b8748b6 100644
--- a/src/libxcrypt/libxcrypt-4.4.28/configure.ac
+++ b/src/libxcrypt/libxcrypt-4.4.28/configure.ac
@@ -46,7 +46,7 @@ esac
 AC_PROG_CPP
 AC_PROG_MAKE_SET
 AC_PROG_LN_S
-zw_PROG_PERL([5.14.0])
+zw_PROG_PERL([5.10.0])
 AC_PATH_PROG([GPG2], [gpg2], [false])
 AC_PATH_PROG([SHA256SUM], [sha256sum], [false])
 
