diff --git a/configure b/configure
index 5074c031..a99e7ce9 100755
--- a/configure
+++ b/configure
@@ -4808,16 +4808,16 @@ $as_echo "no" >&6; }
 fi
 
 
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking whether $PERL is version 5.14.0 or later" >&5
-$as_echo_n "checking whether $PERL is version 5.14.0 or later... " >&6; }
-$as_echo "$as_me:${as_lineno-$LINENO}: $PERL -e 'use v5.14.0;'" >&5
-if "$PERL" -e 'use v5.14.0;' >&5 2>&1; then :
+{ $as_echo "$as_me:${as_lineno-$LINENO}: checking whether $PERL is version 5.10.0 or later" >&5
+$as_echo_n "checking whether $PERL is version 5.10.0 or later... " >&6; }
+$as_echo "$as_me:${as_lineno-$LINENO}: $PERL -e 'use v5.10.0;'" >&5
+if "$PERL" -e 'use v5.10.0;' >&5 2>&1; then :
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
 else
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
 $as_echo "no" >&6; }
-as_fn_error $? "Perl version 5.14.0 or later is required" "$LINENO" 5
+as_fn_error $? "Perl version 5.10.0 or later is required" "$LINENO" 5
 fi
 # Extract the first word of "gpg2", so it can be a program name with args.
 set dummy gpg2; ac_word=$2
