--- Lib/site.py.orig    2020-11-12 09:59:21.576077736 +0000
+++ Lib/site.py 2020-11-12 10:00:01.732005931 +0000
@@ -63,6 +63,18 @@
 import __builtin__
 import traceback

+import warnings
+warnings.filterwarnings(
+    'ignore',
+    category=UserWarning,
+    message='Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.',
+    )
+warnings.filterwarnings(
+    'ignore',
+    category=UserWarning,
+    message='OpenSSL version 1.0.2 is no longer supported by the OpenSSL project, please upgrade. The next version of cryptography will completely remove support for it.',
+    )
+
 # Prefixes for site-packages; add additional prefixes like /usr/local here
 PREFIXES = [sys.prefix, sys.exec_prefix]
 # Enable per user site-packages directory
