diff -ur setup.py setup.py
--- setup.py
+++ setup.py
@@ -2090,6 +2090,7 @@
         include_dirs = []
         extra_compile_args = []
         extra_link_args = []
+        extra_objects = [] 
         sources = ['_ctypes/_ctypes.c',
                    '_ctypes/callbacks.c',
                    '_ctypes/callproc.c',
@@ -2123,6 +2124,7 @@
                         include_dirs=include_dirs,
                         extra_compile_args=extra_compile_args,
                         extra_link_args=extra_link_args,
+                        extra_objects=extra_objects,
                         libraries=[],
                         sources=sources,
                         depends=depends)
@@ -2138,6 +2140,9 @@
             # in /usr/include/ffi
             inc_dirs.append('/usr/include/ffi')
 
+        # On some OS'es there's no libffi, so we use our own.
+        inc_dirs.append('build/libffi')
+
         ffi_inc = [sysconfig.get_config_var("LIBFFI_INCLUDEDIR")]
         if not ffi_inc or ffi_inc[0] == '':
             ffi_inc = find_file('ffi.h', [], inc_dirs)
@@ -2154,15 +2159,21 @@
                     print('Header file {} does not define LIBFFI_H or '
                           'ffi_wrapper_h'.format(ffi_h))
         ffi_lib = None
+        ffi_lib_dirs = lib_dirs[:]
+
+        # On some OS'es there's no libffi, so we use our own.
+        ffi_lib_dirs.append('build/libffi')
+
         if ffi_inc is not None:
             for lib_name in ('ffi_convenience', 'ffi_pic', 'ffi'):
-                if (self.compiler.find_library_file(lib_dirs, lib_name)):
+                if (self.compiler.find_library_file(ffi_lib_dirs, lib_name)):
                     ffi_lib = lib_name
                     break
 
         if ffi_inc and ffi_lib:
             ext.include_dirs.extend(ffi_inc)
-            ext.libraries.append(ffi_lib)
+            # On some OS'es there's no libffi, so we use our own.
+            ext.extra_objects.append('build/libffi/libffi.a')
             self.use_system_libffi = True
 
 
