diff -ur setup.py setup.py
--- setup.py
+++ setup.py
@@ -807,10 +811,23 @@
                              )
         if ssl_incs is not None:
             krb5_h = find_file('krb5.h', inc_dirs,
-                               ['/usr/kerberos/include'])
+                               ['/usr/kerberos/include',
+                                '/usr/include/kerberosv5/'
+                               ])
             if krb5_h:
                 ssl_incs += krb5_h
-        ssl_libs = find_library_file(self.compiler, 'ssl',lib_dirs,
+        # On Solaris 10 the OpenSSL libs live in /usr/sfw/lib.
+        if host_platform == 'sunos5':
+            # Determine if we need to set the path to 64bit OpenSSL libs.
+            arch_env = os.environ.get('ARCH')
+            if '64' in arch_env:
+                ssl_libs = find_library_file(self.compiler, 'ssl',lib_dirs,
+                                         [ '/usr/sfw/lib/64' ] )
+            else:
+                ssl_libs = find_library_file(self.compiler, 'ssl',lib_dirs,
+                                         [ '/usr/sfw/lib' ] )
+        else:
+            ssl_libs = find_library_file(self.compiler, 'ssl',lib_dirs,
                                      ['/usr/local/ssl/lib',
                                       '/usr/contrib/ssl/lib/'
                                      ] )
